app/src/main/java/com/example/lhi06/residue/FindAlbumActivity.java

```diff
@@ -22,12 +24,14 @@
 
 public class FindAlbumActivity extends AppCompatActivity {
 
+    public final static String ALBUM_EXTRA = "com.example.lhi06.residue.ALBUM_EXTRA";
+
     private AlbumAdapter adapter;
     private ItunesService service;
 
 
-    @BindView(R.id.recycleView) RecyclerView recyclerView;
-    @BindView(R.id.request) EditText editText;
+    @BindView(R.id.recycleview_albums_findalbum) RecyclerView recyclerView;
+    @BindView(R.id.edittext_request_findalbum) EditText editText;
```

Comment:
request isn't very suggestive. Is this not an edit text for the artist name? If so I would rename the id to `edittext_artist_name` and rename the field to `artistNameEditText`